test/library/packages/UnitTest/test_locales.chpl

```diff
@@ -0,0 +1,25 @@
+use UnitTest;
+
+// This test requires 8 locales
+proc s1(test: Test) throws {
+  test.addNumLocales(16);
+}
+
+// This test can run with 2-4 locales
+proc s2(test: Test) throws {
+  test.maxLocales(4);
+  test.minLocales(2);
+}
+
+// This test can run with 8 or 16 locales
+proc s3(test: Test) throws {
+  test.addNumLocales(8,16);
+}
+
+// This test has min locales >  max locales
+proc s4(test: Test) throws {
+  test.maxLocales(5);
+  test.minLocales(10);
+}
+
+UnitTest.runTest(s1,s2,s3,s4);
```

Comment:
I don't see a check for `maxLocales < 1`.  It would also be good to check:
- addNumlocales multiple times (as I mentioned above)
- addNumlocales where the locales are not specified in order (e.g. `test.addNumLocales(16, 8)`)
- max and minLocales on their own
- other test cases I'm not thinking of?  There might have been one or two other things you checked in the code itself but didn't add a test for, but I don't remember.

Reply:
I have added these cases.